shell/platform/android/image_external_texture_gl.h

```diff
@@ -30,14 +31,24 @@ class ImageExternalTextureGL : public ImageExternalTexture {
  protected:
   void Attach(PaintContext& context) override;
   void Detach() override;
+  void ProcessFrame(PaintContext& context, const SkRect& bounds) override;
+
+  virtual sk_sp<flutter::DlImage> CreateDlImage(
+      PaintContext& context,
+      const SkRect& bounds,
+      uint64_t id,
+      impeller::UniqueEGLImageKHR&& egl_image) = 0;
 
-  // Returns true if a new image was acquired and android_image_ and egl_image_
-  // were updated.
-  bool MaybeSwapImages();
   impeller::UniqueEGLImageKHR CreateEGLImage(AHardwareBuffer* buffer);
 
   fml::jni::ScopedJavaGlobalRef<jobject> android_image_;
-  impeller::UniqueEGLImageKHR egl_image_;
+
+  struct GlEntry {
```

Comment:
I wish we could inline this into the LRU struct and not keep the separate map here but I can't think of a clean way to do it without messing up the _vk variants.